src/coreclr/jit/emitxarch.cpp

```diff
@@ -16275,6 +16275,13 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
             break;
         }
 
+        case INS_serialize:
+        {
+            result.insLatency    = PERFSCORE_LATENCY_140C;
```

Comment:
@tannergooding do you have a recommendation for a perfscore for `serialize`? `serialize` will take as long as it takes to flush the instruction pipeline, i.e., its latency/throughput is dependent on the instructions in the pipeline. I can try to get a minimum latency/throughput, or we could try a worse case upper bound?

Reply:
We've based our previous numbers based on the official timing numbers for `Skylake` (from `Intel® 64 and IA-32 Architectures Software Developer Manuals` iirc).

This is really just meant to be a rough heuristic and so an estimate or average case should be sufficient.

Reply:
We are looking to get these numbers internally. For the time being, I have set both to be 1 cycle. Can we update these values once we have the numbers?

Reply:
Yes, provided an issue is logged tracking it being updated.

Could we set it to something slightly more meaningful than `1` in the meantime? I'd expect this to at least be as expensive as a fencing operation (`lfence`, `mfence`, `sfence`).

Reply:
I mirrored what was done for `mfence`, and set it to 50 cycles (more expensive than `mfence`, less expensive than `cpuid` which is ~100 cycles per the manual. 

This seem ok? I will open up an issue now.

Reply:
Seems great. Thanks!